### PR TITLE
Milestone 5 - Full website

### PR DESCRIPTION
**Pull request by mukhammadrizooff**
# In this branch I:
- ### I create a website consisting of 3 pages: Home, Calculator, and Quote.
- ### I add my own styles to improve the look and feel.
  - #### In the lesson about styling, I learned several ways to add styles in a React application:
    - React basic styling (inline styles, importing CSS and pre-processed CSS files).
    - CSS modules.
    - Styled components.
  - #### In this project, I use **React basic styling**.
- ### For every page, I create a *route*.
- ### My navigation links for all 3 routes are displayed on each page.
- ### I use [React Router](https://reactrouter.com/) to make the routes work.

### CODE REVIEW

**Comment by mukhammadrizooff** [comment]
### My project was not deployed because my `gh-pages` branch is not working, I apologize for that and will try to resolve the issue as soon as possible. Thank you very much for your service!

File: src/components/home.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import './pages.css';
+
+function Home() {
+  const par = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
+
```
Comment (mmsesay) [inline src/components/home.js:5]: - [OPTIONAL] The font size is a bit smaller. I will suggest you increase it a bit.

Your section home:

a1

**Review by mmsesay — APPROVED** [review]
Hi @mukhammadrizooff, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/home.js (hunk 1 above)
Comment (mukhammadrizooff) [reply src/components/home.js:5]: - Thank you I will do  it.

**Pull request merged**